CONTRIBUTING.md

```diff
@@ -167,50 +204,104 @@ make license-check
+4. **Is there enough capacity on the core team to support a community contributor?**<br />We don't expect contributors to implement the entire feature in isolation, we actively participate with planning, reviews and writing documentation as needed.
+5. **Does this feature enable someone to do something new with OpenTofu they were not able to do before?**<br />We prioritize work based on community input and need. An issue with a large number of reactions is more likely to make it into the accepted phase, but if a viable workaround or tool exists, the feature is less likely to be accepted. If a feature is just the integration of a cool technology, but doesn't solve any problems for a large number of people, it will be rejected.
 
-For example, execute the command to run integration tests with s3 backend:
+Depending on the core team's review, a feature request can have the following outcomes:
 
-```commandline
-make test-s3
-```
+1. **The feature is accepted for development by the core team.** This means that the core team will schedule it for an upcoming release and develop it.
+2. **The feature is accepted and open for community contributions.** The core team adds a `help wanted` label and waits for community volunteers to develop it.
+3. **More information is needed.** The core team will either add questions in comments, or when there is a deep technical issue to be resolved, call for an RFC to detail a possible implementation.
+4. **More community input is needed.** When an issue is, on its surface, valuable, but there is no track record of a large portion of the community needing it, the core team adds the `needs community input` label. If you are interested in the feature and would like to use it, please add a reaction to the issue and add a description on specifically what problem it would solve for you in a comment.
+5. **The feature is rejected.** If based on the criteria above it is not feasible to implement the feature, the core team closes the issue with an explanation why it is being closed.
+6. **The feature is referred to the Technical Steering Committee**. If the feature requires the commitment of a larger amount of core developer time, has legal implications, or otherwise requires leadership attention, the core team adds the feature to the agenda of the Technical Leadership Committee. Once decided, the TSC records the decision in the [TSC_SUMMARY.md](TSC_SUMMARY.md) file.
 
-## Generated Code
+---
 
-Some files in the OpenTofu CLI codebase are generated. In most cases, we update these using `go generate`, which is the standard way to encapsulate code generation steps in a Go codebase.
+### What do the labels mean?
```

Comment:
It's a bit of a nit pick maybe, but would come in handy for navigation:
It would be helpful to hyperlink those labels, such as [accepted](https://github.com/opentofu/opentofu/issues?q=is%3Aopen+is%3Aissue+label%3Aaccepted), [pending-steering-committee-decision](https://github.com/opentofu/opentofu/issues?q=is%3Aopen+is%3Aissue+label%3Apending-steering-committee-decision) etc.